client/a8c-for-agencies/components/layout/index.tsx

```diff
@@ -24,6 +25,13 @@ export default function Layout( {
 	compact,
 	sidebarNavigation,
 }: Props ) {
+	const hasLayoutColumns = React.Children.toArray( children ).some(
+		( child ) => React.isValidElement( child ) && child.type === LayoutColumn
+	);
+	const layoutContainerClassname = ! hasLayoutColumns
+		? 'a4a-layout__container'
+		: 'a4a-layout-with-columns__container';
```

Comment:
[nitpicking] Generally, we would use a positive condition as it is more easier to read than a negative condition.

```suggestion
	const layoutContainerClassname = hasLayoutColumns
		? 'a4a-layout-with-columns__container'
         : 'a4a-layout__container';
```